docs/content/docs/dev/datastream/fault-tolerance/state.md

```diff
@@ -395,11 +395,17 @@ The update type configures when the state TTL is refreshed (by default `OnCreate
 
  - `StateTtlConfig.UpdateType.OnCreateAndWrite` - only on creation and write access
  - `StateTtlConfig.UpdateType.OnReadAndWrite` - also on read access
+
+    (**Notes:** If you set the state visibility to `StateTtlConfig.StateVisibility.ReturnExpiredIfNotCleanedUp`
```

Comment:
It would be great to also add explain a bit on why it will cause performance loss.